pkg/devfile/adapters/docker/component/adapter.go

```
@@ -271,3 +275,38 @@ func (a Adapter) Delete(labels map[string]string) error {
 	return nil
 
 }
+
+func (a Adapter) Log(follow, debug bool) (io.ReadCloser, error) {
+
+	exists, _ := utils.ComponentExists(a.Client, a.Devfile.Data, a.ComponentName)
```

we shouldn't ignore the err.